main/modes/games/cGrove/Garden/cg_GroveDraw.c

```diff
@@ -496,20 +494,6 @@ void cg_drawGroveTutorial(cGrove_t* cg)
                                 3.0);
             break;
         }
-        case 4:
-        {
-            // Guests
-            drawWsgSimpleScaled(&cg->chowaWSGs[CG_KING_DONUT][CG_CHILD][30],
-                                (TFT_WIDTH - (3 * cg->chowaWSGs[CG_KING_DONUT][CG_CHILD][30].h) - 70) >> 1, 100, 3, 3);
-            drawWsgSimpleScaled(&cg->chowaWSGs[CG_KING_DONUT][CG_CHILD][30],
-                                (TFT_WIDTH - (3 * cg->chowaWSGs[CG_KING_DONUT][CG_CHILD][30].h) + 70) >> 1, 100, 3, 3);
-            break;
-        }
-        case 5:
-        {
-            // Menu
-            break;
-        }
         case 6:
```

Comment:
Is this supposed to be `case 6:` or `case 4:`?

Reply:
Deleted case 4 because guest are no longer a feature, by which I mean, you can't get any. ALl the actual code is still there since rooting it out would take a lot of time. Guest should not be tutorialized.

Reply:
Using the aforementioned `enum` option as switch labels also helps to avoid this pitfall.

Reply:
I get deleting cases 4 & 5, but without changing the old 6 to be the new 4 this is unreachable, right? `cg->grove.tutorialPage` only counts to 5, so might as well delete case 6 too?

Reply:
You are correct, it wouldn't be hit. Updated to 5, since it's supposed to trigger on the last page.

Reply:
There's still a gap at 4, but if that's the way its supposed to work, fine by me!

Reply:
Nothing to draw on page 4, so yeah